src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/AlertSetting/SetAzureRmRecoveryServicesAsrAlertSetting.cs

```diff
@@ -0,0 +1,172 @@
+﻿// ----------------------------------------------------------------------------------
+// 
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System.Threading;
+using Microsoft.Azure.Management.RecoveryServices.SiteRecovery.Models;
+
+namespace Microsoft.Azure.Commands.RecoveryServices.SiteRecovery
+{
+    /// <summary>
+    ///     Sets Azure Site Revcovery Notification / Alert.
+    /// </summary>
+    [Cmdlet(
+        VerbsCommon.Set,
+        "AzureRmRecoveryServicesAsrAlertSetting",
+        DefaultParameterSetName = ASRParameterSets.Set)]
+    [Alias(
+        "Set-ASRNotificationSetting",
+        "Set-AzureRmRecoveryServicesAsrNotificationSetting",
+        "Set-ASRAlertSetting")]
+    [OutputType(typeof(ASRAlertSetting))]
+    public class SetAzureRmRecoveryServicesAsrAlertSetting : SiteRecoveryCmdletBase
+    {
+        /// <summary>
+        ///     Gets or sets EmailSubscriptionOwner .Mail to subscription owner.
+        /// </summary>
+        [Parameter(ParameterSetName = ASRParameterSets.EmailToSubscriptionOwner, Mandatory = true)]
+        public SwitchParameter EmailSubscriptionOwners { get; set; }
```

Comment:
@viverm this should be renamed to `EnableEmailSubscriptionOwners`

Reply:
Changed